spec/webmachine/events_spec.rb

```diff
@@ -48,7 +48,7 @@
 
   describe ".unsubscribe" do
     it "calls the backend" do
-      subscriber = described_class.subscribe('test.event') { }
+      subscriber = described_class.subscribe('test.event', Proc.new { })
 
```

Comment:
If I'm understanding the change to as-notificaitons correctly, I'm guessing this can be reverted once the gem dependency is updated?

Reply:
I hope so, yes – I think this spec was the only place where we were calling it this way.